newIDE/app/src/EventsSheet/EventsTree/Renderers/GroupEvent.js

```diff
@@ -121,7 +122,11 @@ export default class GroupEvent extends React.Component<EventRendererProps, *> {
             })}
             style={{ ...styles.title, color: textColor }}
           >
-            {groupEvent.getName() || '<Enter group name>'}
+            {groupEvent.getName() ? (
+              groupEvent.getName()
+            ) : (
+              <Trans>{`<Enter group name>`}</Trans>
```

Comment:
Same as before, this will be parsed as "{0}", so nothing will be translated.
Instead, try: `<Trans>&lt;Enter group name&gt;</Trans>`

Reply:
It works fine with my code ;)
![image](https://user-images.githubusercontent.com/1670670/103373504-7072e700-4ad5-11eb-87d7-8fba7463b37d.png)

Reply:
Oh right then! I was not aware that jslingui was so smart at parsing these!

Reply:
Confirmed in their source code: https://github.com/lingui/js-lingui/blob/3afaceccb669b59ee2f5b42ee2e138646ccdb79d/packages/macro/test/jsx-trans.ts#L96-L107